src/libstore/ca-specific-schema.sql

```diff
@@ -0,0 +1,13 @@
+-- Extension of the sql schema for content-addressed derivations.
+-- Won't be loaded unless the experimental feature `ca-derivations`
+-- is enabled
+
+create table if not exists Realisations (
```

Comment:
There's indeed a terminology mismatch and we need to settle on one thing.

I really like `Realisation` tbh because these are precisely witnesses that a derivation has been realised (for the definition you give above). But it's indeed a quite overloaded term so if you don't like it we can find something else.

I tend to use `DerivationOutput` only for the “symbolic” `foo.drv!out` because it's also a very overloaded term and it's easy to get lost if we abuse of it.

(also, they aren't really `Derivers` (though they could be used as such) because the intent is to associate derivation outputs (`foo.drv!out`) to their output paths and not the other way around)

TL;DR, I'm fine changing the naming, but for my own sanity (and other's too) I think we need to find clear and distinct names for

1. `foo.drv!out`
2. The relation `(foo.drv!out, /nix/store/123-foo-out)`